custom/opm/reports.py

```diff
@@ -55,183 +43,6 @@
 from .constants import *
 
 
-DATE_FILTER = "date between :startdate and :enddate"
-DATE_FILTER_EXTENDED = """(
-    opened_on <= :enddate AND (
-        closed_on >= :enddate OR
-        closed_on = ''
-        )
-    ) OR (
-    opened_on <= :enddate AND (
-        closed_on >= :startdate or closed_on <= :enddate
-        )
-    )
-"""
-DATE_FILTER_EXTENDED_OPENED = """(
-    opened_on <= :enddate AND (
-        closed_on >= :enddate OR
-        closed_on = ''
-        )
-    )
-"""
-DATE_FILTER_EXTENDED_CLOSED = """(
-    opened_on <= :enddate AND (
-        closed_on <= :enddate AND
-        closed_on != ''
-        )
-    )
-"""
-
-EDD_DOD_FILTER = """
-    (
-       (edd >= :edd_startdate AND edd <= :edd_enddate)
-       OR
-       (dod <= :enddate and dod != '')
-    )
-"""
-
-def ret_val(value):
-    return value
-
-
-class OpmCaseSqlData(SqlData):
```

Comment:
@snopoke I'm not super familiar with `SqlData` - I think removing this will leave an unused table in postgres, is there any additional cleanup I should do?